internal/runners/run/run.go

```diff
@@ -75,7 +75,7 @@ func run(out output.Outputer, subs subshell.SubShell, name string, args []string
 		return fail
 	}
 
-	path := os.Getenv("PATH")
+	var venvPath string
```

Comment:
Can we rename this to `venvExePath` and maybe add a comment explaining what it does. I realize you didn't introduce this but reviewing it it was a stumbling block figuring out what it was used for.